scripts/combatdata.gd

```diff
@@ -3,12 +3,12 @@ extends Node
 
 var enemygrouppools = {
```

Comment:
awareness of all enemies has been altered without condition

Reply:
This was a bug ralph introduced with elif scoutawareness < (enemyawareness + rand_range(-7,7)), does leaving it at -10 harm anything?

Reply:
I suppose that -10 awareness is normally not different from 0 for at least 99% of cases, so it should be safe to leave the 0 to -10 changes. Though "monstergirl" was increased from 0 to 15, which is a meaningful change.

Reply:
Fixed